### PR TITLE
Improved client logging.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Minor improvements to client logging, and brings the client module up to date to use recent improvements in the SearchRequest API.

### CODE REVIEW

File: ga4gh/client.py [hunk 1]
```diff
@@ -12,9 +12,6 @@
 
 import ga4gh.protocol as protocol
 
-# suppress warning about using https without cert verification
```
Comment (dcolligan) [inline ga4gh/client.py]: why take this out?

Comment (dcolligan) [inline ga4gh/client.py]: er, nm, I see it's called in the __init__

File: ga4gh/client.py [hunk 2]
```diff
@@ -38,10 +36,13 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         else:
             logLevel = logging.DEBUG
         self._logger.setLevel(logLevel)
-        handler = logging.StreamHandler()
-        formatter = logging.Formatter('%(name)s: %(message)s')
-        handler.setFormatter(formatter)
-        self._logger.addHandler(handler)
+
+        requestsLog = logging.getLogger("requests.packages.urllib3")
```
Comment (dcolligan) [inline ga4gh/client.py]: Is this really a good idea?  We lose control over formatting our messages differently from the messages that urllib3 wants to emit

Comment (jeromekelleher) [inline ga4gh/client.py]: I was just hacking around here, as it's tricky getting the different loggers working together. This was the simplest way I could find that meant our own logs weren't getting written twice.

Totally open to ideas here if you can see a better way.

Comment (dcolligan) [inline ga4gh/client.py]: why were our logs getting written twice?

Comment (jeromekelleher) [inline ga4gh/client.py]: Dunno - seems like they were getting sent to the requests logger as well as the one we'd instantiated.

Comment (dcolligan) [inline ga4gh/client.py]: Ok, put a TODO here to disentangle our logger from the urllib3 logger, without printing duplicate messages.

Comment (jeromekelleher) [inline ga4gh/client.py]: I've put in a TODO here. I'm not particularly happy with what we've ended up with --- I wish I understood what was actually happening here. It'll do for now though I think.

Comment (dcolligan) [inline ga4gh/client.py]: @jeromekelleher reading this (and other python logging docs) will probably help: https://docs.python.org/2/howto/logging.html#logging-flow

**Pull request merged**